.github/scripts/print-gradle-workers-max.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# We know the gradle daemon takes 1GiB (see gradle.properties)
+DAEMON_BYTES=1073741824
+
+# We'll leave some buffer space for other system resources / processes
+OTHER_BYTES=2147483648
+
+TOTAL_SYSTEM_BYTES="$(free --bytes | grep Mem | awk -F " " '{print $2}')"
+
+# This is accounting for "worst case", assuming every single worker is using the theoretical maximum.
+# Currently, engine/table/build.gradle sets a heap size of 6GiB, so that's the maximum.
+PER_WORKER_BYTES=6442450944
+
+MAX_WORKERS="$(( (TOTAL_SYSTEM_BYTES - DAEMON_BYTES - OTHER_BYTES) / PER_WORKER_BYTES ))"
```

Comment:
This is memory right?
I get that we use this just below, to echo it as a property value.
If we are not going to call this MAX_WORKER_MEM_BYTES, can we at least call it WORKERS_MAX, like the property is blah.blah.blah.workers.max below?
I imagine we didn't pick that property name, so be it, but at least lets not add another permutation on a confusing name...

Reply:
The unit is "units", not "memory". Unit analyses, the formula is:  (bytes) / (bytes / worker unit) -> worker unit